docs/static/jvm.asciidoc

```diff
@@ -4,8 +4,9 @@
 
 {ls} requires one of these versions:
 
-* Java 11
-* Java 17 (default). Check out <<jdk17-upgrade>> for settings info.
+* Java 11.
+* Java 17. Check out <<jdk17-upgrade>> for settings info.
```

Comment:
@robbavey, we'll need to decide what to do with the <<jdk17-upgrade>> section.  Maybe rename it to "Using JDK 21" now that JDK 21 is the default, and updating content in that section accordingly?